src/chrome/content/rules/ImageShack.xml

```diff
@@ -12,19 +13,22 @@
 	<target host="www.imageshack.com" />
 	<target host="api.imageshack.com" />
 	<target host="blog.imageshack.com" />
+	<target host="imagizer.imageshack.com" />
 
 	<target host="imageshack.us" />
-
-	<!-- Show 404 page: -->
-		<test url="http://a.imageshack.us/img254/8219/40789751.png" />
 	<target host="*.imageshack.us" />
+		<test url="http://www.imageshack.us/" />
+		<!-- Show 404 page: -->
+		<test url="http://a.imageshack.us/img254/8219/40789751.png" />	
+		<test url="http://imagizer.imageshack.us/" />
+		<test url="http://post.imageshack.us/" />
 		<test url="http://img5.imageshack.us/img5/133/homeworld.jpg" />
 		<test url="http://img46.imageshack.us/img46/4804/intel1jk8.gif" />
 		<test url="http://img530.imageshack.us/img530/133/homeworld.jpg" />
 
 	<securecookie host=".+" name=".+" />
 
-	<rule from="^http://(www\.|api\.|blog\.)?imageshack\.com/" to="https://$1imageshack.com/" />
+	<rule from="^http://(www\.|api\.|blog\.|imagizer\.)?imageshack\.com/" to="https://$1imageshack.com/" />
 
 	<rule from="^http://(www\.|a\.|imagizer\.|post\.)?imageshack\.us/" to="https://$1imageshack.us/" />
 
```

Comment:
Can we replace all redirect rules with one simple `<rule from="^http:" to="https:" />`?

Reply:
Only if we can make sure that all broken imageshack.us subdomains are known and there are exclude patterns matching all of them. Otherwise the *.imageshack.us target will cause problems.

Reply:
I'm not aware of any other domains than the listed ones, but I'm also fine with keeping it for now.